readinglist/resource.py

```diff
@@ -130,7 +130,11 @@ def _extract_filters(self, queryparams):
             if param in self.known_fields:
                 filters.append((param, value, '=='))
             if param == '_since':
-                filters.append((self.modified_field, value, '>='))
+                if isinstance(value, six.integer_types + (float,)):
+                    filters.append((self.modified_field, value, '>='))
```

Comment:
the only advantage I about using constants is to have an exhaustive list somewhere.
(think backend implementation https://github.com/mozilla-services/readinglist/blob/safety-check-for-filter-values/readinglist/backend/memory.py#L55-L62)